blog/next.config.js

```diff
@@ -3,11 +3,21 @@ const withCustomBabelConfig = require("next-plugin-custom-babel-config");
 const withTypescript = require("@zeit/next-typescript");
 const withTranspileModules = require("next-plugin-transpile-modules");
 
-module.exports = withCustomBabelConfig(
+// https://github.com/wellcometrust/next-plugin-transpile-modules/issues/11
+function fixExternalsPlugin(config = {}) {
+  const { webpack } = config
+  config.webpack = (config, ...rest) => {
+    config.externals = config.externals || []
+    return webpack(config, ...rest)
+  }
+  return config
+}
+
+module.exports = fixExternalsPlugin(withCustomBabelConfig(
```

Comment:
Can you nest the plugins with newlines as per the other plugins? Keeps them readable.

Reply:
Refactored with use of next-compose-modules.